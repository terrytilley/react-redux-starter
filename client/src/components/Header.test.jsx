import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Header from './Header';

describe('<Header />', () => {
  it('should render component', () => {
    const wrapper = shallow(<Header />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
