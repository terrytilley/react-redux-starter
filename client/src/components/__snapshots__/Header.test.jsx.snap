// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render component 1`] = `
<ul>
  <li>
    <Link
      replace={false}
      to="/"
    >
      Home
    </Link>
  </li>
  <li>
    <Link
      replace={false}
      to="/about"
    >
      About
    </Link>
  </li>
</ul>
`;
